docs/manuals/gui/binding.md

```diff
@@ -20,105 +21,179 @@ The variable `x` contains the value <|{x}|>.
 Taipy pages, binding visual elements to local variables. That allows for
 a clearer organization of the code, where application variables used in
 only a few pages can sit next to the page definition.
 
-[TODO EXAMPLE]
+When Taipy finds a variable in a page, it first tries to locate it in the module
+where this page is defined. If the variable can not be found in the page module,
+then the variable is sought in the *\_\_main\_\_* module (typically, where the
+`Gui^` instance is created).
 
-## Expressions
+This mechanisms allows to create pages on their own module, with local variables
+that the rest of the application may not be aware of. Global variables (the ones
+declared in the *\_\_main\_\_* module), on the other hand, can be used in all
+page content definitions. The module where the page is defined does not need to
+import the global variables, if they are not used in the Python code of the module.
 
-Values that you can use in controls and blocks can be more than raw variable values.
-You can create complete expressions, just like you would use
-in the _f-string_ feature (available since Python 3).
+!!! example "Example"
 
-In the code above, you could replace `<|{x}|>` by `<|{x*2}|>`, and the double of _x_
-will be displayed on your page.
+    Say you want to create a page that represents some evalution of an
+    expression using a global variable as well as some local parameter.
 
-!!! Note "Arbitrary expressions"
-        You can create complex expressions such as `|{x} and {y}|` to concatenate
-        two variable values, or whatever your imagination and application requirements are.
+    This local parameter is useless for the rest of the application, so
+    we store it in a local variable, in the module where the page is
+    defined.<br/>
+    We create a _pages_ package, where we can store the file `page.py` which is
+    the module file where the page would be declared:
 
-!!! Note "Formatting"
-        F-string formatting is also available in property value expressions:
+    ```py linenums="1"
+    # File: pages/page.py
+    from taipy.gui import Markdown
+
+    page = Markdown("""# Expression evaluation
+    Expression value: <|{base_value + local_value}|>
+    """)
+
+    local_value=0
+    ```
+
+    Note that the page definition uses the variable _base_value_ (in line 5) that is
+    unknown to this module.
+
+    The page also references the variable _local_value_ (line 8), which is local to
+    this module.
+
+    Now let's create the whole application, adding a home page, and reusing
+    the page that we just created above. That would be done in the file `main.py`
+    defining the *\_\_main\_\_* module of our application:
+
+    ```py  linenums="1"
+    # File: main.py
+    from taipy.gui import Gui, Markdown
+    from pages.page import page
+
+    navigation = [("/home", "Home"), ("/page", "Expression")]
+    root_page="""# Expression
+    <|navbar|lov={navigation}|>
+    """
+
+    root_page="""# Home page
```

Comment:
I suppose this is home_page instead of root_page ?

Reply:
Fixed. Thanks.